courses/models.py

```diff
@@ -1,3 +1,53 @@
 from django.db import models
+from accounts.models import CustomUser
 
-# Create your models here.
+class Curriculum(models.Model):
+    name = models.CharField(max_length=255, verbose_name='커리큘럼 이름')
+    description = models.TextField(verbose_name='설명')
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+class Course(models.Model):
+    curriculum = models.ForeignKey(Curriculum, on_delete=models.CASCADE, related_name='courses')
```

Comment:
Curriculum 이 삭제 하더라도 Course는 삭제 하면 안될 것 같아요.
```suggestion
    curriculum = models.ForeignKey(Curriculum, on_delete=models.SET_NULL, null=True, related_name='courses')
```